pkg/download/automation/automation.go

```diff
@@ -102,6 +111,12 @@ func (d *Downloader) Download(projectName string, automationTypes ...config.Auto
 	return configsPerType, nil
 }
 
+func escapeJinjaTemplates(src []byte) ([]byte, error) {
+	var prettyJSON bytes.Buffer
+	err := json.Indent(&prettyJSON, src, "", "\t")
```

Comment:
out of curiosity: why indent with tabs rather than spaces? Probably doesn't make any difference, but later on the JSON template will be re-indented with spaces.

Reply:
It might be more consistent if someone just looks for "Indent" via search. 
I don't think it has any performance impact if we'd indent in a way that does not actually need to be re-indented. 

Fine for me to leave as is